comictalker/talkers/comicvine.py

```diff
@@ -203,7 +216,11 @@ def parse_settings(self, settings: dict[str, Any]) -> dict[str, Any]:
         self.remove_html_tables = settings["cv_remove_html_tables"]
         return settings
 
-    def check_api_key(self, key: str, url: str) -> bool:
+    def check_api_key(
+        self,
+        url: str,
+        key: str,
+    ) -> str:
```

Comment:
Is there a reason for this to be expanded?
```suggestion
    def check_api_key(self, url: str, key: str) -> str:
```